modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/FragmentPreview.es.js

```diff
@@ -160,7 +160,7 @@ class FragmentPreview extends PortletBase {
 		if (!this._loading) {
 			this._loading = true;
 
-			this.fetch(this.urls.render, {
+			fetch(this.urls.render, {
```

Comment:
This code would fail. The second argument of `Liferay.Util.fetch` is not the same as `PortletBase.fetch`. The former is init object, the latter object literal that is to be transferred to `body` option.